code/modules/ruins/spaceruin_code/forgottenship.dm

```diff
@@ -33,18 +33,45 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
 	info = "Greetings, operatives. You are assigned to SCSBC-12(Syndicate Cyber Sun Battle Cruiser 12) to protect our high-ranking officer while he is on his way to next outpost. While you are travelling, he is the captain of this ship and <b>you must</b> obey his orders.<br><br>Remember, disobeying high-ranking officer orders is a reason for termination."
 
 ///////////	forgottenship items
+
 /obj/item/disk/surgery/forgottenship
 	name = "Advanced Surgery Disk"
 	desc = "A disk that contains advanced surgery procedures, must be loaded into an Operating Console."
 	surgeries = list(/datum/surgery/advanced/lobotomy, /datum/surgery/advanced/bioware/vein_threading, /datum/surgery/advanced/bioware/nerve_splicing)
 
 /obj/structure/fluff/empty_sleeper/syndicate/captain
 	icon_state = "sleeper_s-open"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 
 /obj/structure/fluff/empty_sleeper/syndicate/captain/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/gps, "Old Encrypted Signal")
 
+/obj/item/storage/box/firingpins/syndicate
+	name = "box of syndicate firing pins"
+	desc = "A box full of special syndicate firing pins which allow only syndicate operatives to use weapons with those firing pins."
+	icon_state = "secbox"
+	illustration = "firingpin"
```

Comment:
```suggestion
```

Reply:
Oh, the parent object has those already, right?

Reply:
It does